components/ide/jetbrains/backend-plugin/settings.gradle.kts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the GNU Affero General Public License (AGPL).
 // See License.AGPL.txt in the project root for license information.
 
-rootProject.name = "jetbrains-backend-plugin"
+rootProject.name = "gitpod-remote"
```

Comment:
We need to change the name here because new platform plugin will treat it as build result name